WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -820,9 +820,6 @@
 		5783FB3F25D7369F00B9984B /* WooAnalyticsEventPropertyType.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5783FB3E25D7369F00B9984B /* WooAnalyticsEventPropertyType.swift */; };
 		57896D6625362B0C000E8C4D /* TitleAndEditableValueTableViewCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57896D6525362B0C000E8C4D /* TitleAndEditableValueTableViewCellViewModel.swift */; };
 		5791FB4224EC834300117FD6 /* MainTabViewModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5791FB4124EC834300117FD6 /* MainTabViewModelTests.swift */; };
-		5795F22C23E26A8D00F6707C /* OrderSearchStarterViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5795F22B23E26A8D00F6707C /* OrderSearchStarterViewController.xib */; };
```

Comment:
What happened here? I see many deletions in the project file. I assume these were accidental?

Reply:
I have no idea what happened here, but none of this classes are used in the code anywhere, and the files are not on disk. 

I am going to replicate the PR in a separate branch, and see what happens

Reply:
Okay, if I branch out from trunk and remove Site+Woo.swift from the project, the same changes happen again. Which does not make any sense